content/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -0,0 +1,155 @@
+
+### When querying data
+When querying data from InfluxDB 2.0 using the [`/query` compatibility endpoint](/v2.0/reference/api/influxdb-1x/query/),
+the DBRP mapping service uses the specified database and retention policy to:
+
+Check for a DBRP mapping that matches the database and retention policy.
+
+- If a DBRP mapping exists, query data from the mapped bucket.
+- If no mapping exists, return an error.
+
+##### If no retention policy is specified:
+1. Check for the specified database in the DBRP mapping service.
+  - If the database doesn't exists, return an error.
+  - If database exists, get the default retention policy for the specified database, and then:
+2. Continue the [normal DBRP mapping behavior](#when-querying-data).
+
+## Automatically create DBRP mappings
+
+- [Create a new bucket and DBRP mapping](#create-a-new-bucket-and-dbrp-mapping)
+- [Automatically map to existing buckets](#automatically-map-to-existing-buckets)
+
+{{% warn %}}
+Only the `/write` compatibility endpoint automatically creates DBRP mappings.
+The `/query` compatibility endpoint will return an error if querying a database
+and retention policy combination that is not mapped to a bucket.
+{{% /warn %}}
+
+### Create a new bucket and DBRP mapping
+When using the `/write` compatibility endpoint to write data to InfluxDB 2.0
+and no DBRP mapping exists for the specified database and retention policy and
+no bucket exists that matches the `<database>/<retention-policy>` naming convention,
+InfluxDB **creates a new bucket** using the `<database>/<retention-policy>` naming convention.
+
+**For example:**
+
+```sh
+curl -XPOST http://localhost:9999/write?db=mydb&rp=myrp \
+  -H "Authorization: Token YourAuthToken" \
+  --data-binary "measurement,host=host1 field1=2i,field2=2.0 1577836800000000000"
+```
+
+If no DBRP mapping exists for the `mydb` and `myrp` combination **and** no bucket
+exists with the name `mydb/myrp`, InfluxDB automatically creates a new bucket named
+`mydb/myrp` and writes data to the new bucket.
+
+_See [DBRP behavior when writing data](#when-writing-data)._
+
+### Automatically map to existing buckets
+When using the `/write` compatibility endpoint to write data to InfluxDB 2.0
+and no DBRP mapping exists for the specified database and retention policy, but
+a bucket matching the `<database>/<retention-policy>` naming convention does exist,
+InfluxDB **adds a DBRP mapping for the existing bucket**.
+
+**For example:**
+
+```sh
+curl -XPOST http://localhost:9999/write?db=mydb&rp=myrp \
+  -H "Authorization: Token YourAuthToken" \
+  --data-binary "measurement,host=host1 field1=2i,field2=2.0 1577836800000000000"
+```
+
+If no DBRP mapping exists for the `mydb` and `myrp` combination **but** there is
+a `mydb/myrp` bucket, InfluxDB automatically creates the DBRP mapping and writes
+data to the `mydb/myrp` bucket.
+
+_See [DBRP behavior when writing data](#when-writing-data)._
+
+## Manually create and manage DBRP mappings
```

Comment:
When would it make sense to "manually" create mapped buckets instead of "automatically" creating them? If I'm trying to map all my db/rps to buckets and have a bunch of db/rp combos, would it be easier to send a write request to each db/rp combo to create buckets. Would "manually" creating DBRP mappings require me to create a bucket (to obtain an ID) beforehand?

Reply:
So the InfluxDB 1.x api requires a db/rp combo to query. You may already have a 2.0 bucket that you just want to write data to, but it may not match the db/rp naming convention. You can manually create the mapping so that all db/rp requests get routed to bucket1.

Reply:
And what if you're not trying to query, but just installed v2 and want to move all your v1 db/rps to buckets 1,2,3...what's easiest way to write that data over? (and are the requirements different if you do the "mappings" "manually" or "automatically"?

Reply:
I depends on how you do it. This is probably better left to a 1.x to 2.0 migration guide. I think we're still defining the migration path, so I don't know the best answer.

Reply:
Make sense, thanks Scott. Think I got the details of how mapping works, just not the big picture of how to migration